src/main/java/pages/HomeLoggedPage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import controls.Button;
+import controls.Text;
+import core.Driver;
+import core.PropertiesContainer;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class HomeLoggedPage extends BasePage {
+    private Button button_SignOut() {
+        return Button.byCss("li button[type='submit']");
+    }
+    private Button button_EditingProfile() {
+        return Button.byCss("a[href='/" +PropertiesContainer.get("test.login")+ "']");
+    }
+    private Button button_Profile() {
+        return Button.byCss("summary[class^='HeaderNavlink']>img[alt^='@']");
+    }
+    private Button button_CreateRepo() {
+        return Button.byCss("a[href='/new'][class^='btn'][data-ga-click^='Hello World']");
+    }
+    private Text text_SearchField() {
+        return Text.byCss("input[class^='form-control']");
+    }
+
+    public RepoCreationPage goToRepoCreation() {
+        button_CreateRepo().click();
+        RepoCreationPage repoCreationPage = new RepoCreationPage();
+        try {
+            repoCreationPage.verifyRepoCreation();
+        } catch (Exception e) {
+            Assert.fail(e.getMessage());
+        }
+        return repoCreationPage;
+    }
+
+    public ProfilePage goToProfilePage() {
+        button_Profile().click();
```

Comment:
what if profile btn is already clicked? implement a check that menu is not opened yet